src/Logging.Legacy/LegacyOrleansLogger.cs

```diff
@@ -10,8 +10,8 @@
 namespace Microsoft.Orleans.Logging.Legacy
 {
     /// <summary>
-    /// LegacyOrleansLogger supports legacy orleans logging features, including <see cref="ILogConsumer"/>, <see cref="ICloseableLogConsumer">,
-    /// <see cref="IFlushableLogConsumer">, <see cref="Severity">. 
+    /// LegacyOrleansLogger supports legacy orleans logging features, including <see cref="ILogConsumer"/>, <see cref="ICloseableLogConsumer"/>,
```

Comment:
'Orleans'

Reply:
you want me to make the first letter uppercase? please be more clear on your comments